src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -799,6 +799,196 @@ jsBackend.mediaLibraryHelper.group =
     }
 };
 
+jsBackend.mediaLibraryHelper.cropper =
+{
+    cropperQueue: [],
+    isCropping: false,
+
+    passToCropper: function(resizeInfo, resolve, reject) {
+        jsBackend.mediaLibraryHelper.cropper.cropperQueue.push({
+            'resizeInfo': resizeInfo,
+            'resolve': resolve,
+            'reject': reject,
+        });
+
+        // If the cropper is already handling the queue we don't need to start it a second time.
```

Comment:
Starting comments with a capital seems more logic to me.
But indeed most of the comments in this mediaLibraryHelper.js (created by me) have inconsistent first letter case...